health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/SidePanel.js

```
@@ -0,0 +1,243 @@
+          width: isOpen ? sliderWidth : type === "dynamic" ? defaultClosedWidth || 64 : 0,
+          zIndex: "1000000",
+        }}
+      >
+        {type === "dynamic" && (
+          <div
+            className={`slider-handler ${type || ""} ${position || ""}`}
+            onClick={toggleSlider}
+            onMouseDown={handleMouseDown}
+            style={{ cursor: isDraggable && isOpen ? "ew-resize" : "pointer" }}
+          >
+            {!hideScrollIcon && <div className="scroll-bar"></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)